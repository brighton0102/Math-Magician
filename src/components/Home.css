.homepage-title {
  font-size: 30px;
}

.description {
  font-size: 20px;
}
