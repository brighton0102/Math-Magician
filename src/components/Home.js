import React from 'react';
import './Home.css';

const Home = () => (
  <div>
    <p className="homepage-title">Welcome to Our Page</p>
    <p className="description">Explore our calculator and get inspired by our daily qoutes</p>
  </div>
);

export default Home;
