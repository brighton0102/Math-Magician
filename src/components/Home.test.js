import React from 'react';
import renderer from 'react-test-renderer';
import Home from './Home';

test('Home snapshot test', () => {
  const component = renderer.create(<Home />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
