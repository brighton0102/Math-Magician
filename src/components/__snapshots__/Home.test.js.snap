// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home snapshot test 1`] = `
<div>
  <p
    className="homepage-title"
  >
    Welcome to Our Page
  </p>
  <p
    className="description"
  >
    Explore our calculator and get inspired by our daily qoutes
  </p>
</div>
`;
